hub/core/storage/local.py

```diff
@@ -156,3 +157,9 @@ def _check_is_file(self, path: str):
         if os.path.isdir(full_path):
             raise DirectoryAtPathException
         return full_path
+
+    def clear(self):
+        """Clears the root of the StorageProvider"""
```

Comment:
More threatening message here. This is basically the equivalent of `rm -r *`, we don't want them to think the clearing is reversible or superficial